bricksrc/system.py

```diff
@@ -59,6 +59,9 @@
                             },
                         },
                     },
+                    "Condenser_Water_System": {
+                        "tags": [TAG.Condenser, TAG.Water, TAG.System]
+                    },
                 },
             },
```

Comment:
I feel we have to add `HVAC_System` and change the existing `HVAC` to `HVAC_Equipment` as mentioned in another comment.